jetty-server/src/main/java/org/eclipse/jetty/server/ForwardedRequestCustomizer.java

```diff
@@ -380,9 +381,16 @@ public void customize(Connector connector, HttpConfiguration config, Request req
         {
             for (HttpField field : httpFields)
             {
-                MethodHandle handle = _handles.get(field.getName());
-                if (handle != null)
-                    handle.invoke(forwarded, field);
+                try
+                {
+                    MethodHandle handle = _handles.get(field.getName());
+                    if (handle != null)
+                        handle.invoke(forwarded, field);
+                }
+                catch (Throwable t)
+                {
+                    onError(field, t);
```

Comment:
Sorry but... thinking about this more, can we avoid the double try blocks.  onError has no declared exceptions, so whatever it throws can be thrown from this method.  If it doesn't throw then we continue.  Why the need to wrap with a RuntimeException?

Reply:
oops.. you're right, fixed.